release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -139,6 +139,13 @@ When the Node Tuning Operator recognizes the performance conditions to activate
 
 With this release, the resource limits for the descheduler operand are removed. This enables the descheduler to be used for large clusters with many nodes and pods without failing due to out-of-memory errors.
 
+[id="ocp-4-14-nodes-pod-topology-constraints-matchlabelkeys"]
+==== Pod topology spread constraints matchLabelKeys parameter is now generally available
+
+The `matchLabelKeys` parameter for configuring pod topology spread constraints is now generally available in {product-title} 4.14. Previously, the parameter was available as a Technology Preview feature by enabling the `TechPreviewNoUpgrade` feature set. The `matchLabelKeys` parameter takes list of pod label keys to select the pods to calculate spreading over.
```

Comment:
```suggestion
The `matchLabelKeys` parameter for configuring pod topology spread constraints is now generally available in {product-title} 4.14. Previously, the parameter was available as a Technology Preview feature by enabling the `TechPreviewNoUpgrade` feature set. The `matchLabelKeys` parameter takes a list of pod label keys to select the pods to calculate spreading over.
```

Reply:
Pushed that update, but will wait to hear back from Alex before I do anything else!